code/obj/item/clothing/gimmick.dm

```diff
@@ -1158,6 +1159,8 @@
 						playsound(src.loc, "sound/items/coindrop.ogg", 50, 1, null, 2)
 						sleep(rand(2,5))
 
+/obj/item/clothing/gloves/ring/concrete
```

Comment:
still bad name, no need for this to be abstract, also ATOMIZE PRs

Reply:
name is bad, reason is not- `ABSTRACT_TYPE(/obj/item/clothing/gloves/ring/wizard)` has been established for months. anyway i fixed it by making the one instantiation of `ring` in the underwater ghost house prefab a `ring/gold`.